src/coreclr/src/gc/gc.cpp

```diff
@@ -3576,16 +3985,39 @@ heap_segment* seg_mapping_table_segment_of (uint8_t* o)
 
     size_t index = (size_t)o >> gc_heap::min_segment_size_shr;
     seg_mapping* entry = &seg_mapping_table[index];
-
+    heap_segment* seg = 0;
```

Comment:
Nit: you could keep heap_segment * seg = ... in both places, no?

Reply:
you mean like this: 
```
#ifdef USE_REGIONS
...
heap_segment* seg = (heap_segment*)&seg_mapping_table[index];
#else //USE_REGIONS
    heap_segment* seg = ((o > entry->boundary) ? entry->seg1 : entry->seg0);
#endif //USE_REGIONS
```
? 

I can do that.

Reply:
Yes, that's what I meant.